src/main/java/org/spring/FullVersion/FamilyService.java

```diff
@@ -0,0 +1,112 @@
+package org.spring.FullVersion;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class FamilyService {
+
+    private final FamilyDao familyDao;
+
+    public FamilyService(FamilyDao familyDao) {
+        this.familyDao = familyDao;
+    }
+
+    public List<Family> getAllFamilies() {
+        return familyDao.getAllFamilies();
+    }
+
+    public void displayAllFamilies() {
+        List<Family> allFamilies = familyDao.getAllFamilies();
+        for (int i = 0; i < allFamilies.size(); i++) {
+            System.out.println(i + ": " + allFamilies.get(i));
```

Comment:
I think no need show with indexes, you can use streams instead traditional for loop